app/controllers/services_controller.rb

```diff
@@ -13,4 +13,10 @@ def create
       render json: ErrorsSerializer.new(service).attributes, status: :unprocessable_entity
     end
   end
+
+  def collaborators
```

Comment:
i defo think we need to have the idea of collaborators, however in this instance this method is returning services for an individual user right? 

we probably should have an endpoint that returns a list of collaborators for a given service at some point

Reply:
Yes, returning services. We had difficulty on the naming 😅

Reply:
ok, maybe `services_for_user` or `user_services`, something like that?

Reply:
`services_from_user` maybe?

Reply:
hmm, the `from` implies the user is sending the services to the metadata api. i'll leave it up to you peeps though 😆

Reply:
We changed to `services_for_user` to be the same naming as the documentation